packages/web/src/components/result/ReactiveList.js

```diff
@@ -176,40 +176,40 @@ class ReactiveList extends Component {
 			});
 		}
 
-		if (this.props.stream !== nextProps.stream) {
-			this.props.setStreaming(nextProps.componentId, nextProps.stream);
+		if (prevProps.stream !== this.props.stream) {
+			prevProps.setStreaming(this.props.componentId, this.props.stream);
```

Comment:
I feel you have simply replaced `this.props` -> `prevProps` and `nextProps` -> `this.props`. This won't work. Please test all the edge cases properly.